internal/git/gitignore.go

```diff
@@ -94,11 +94,17 @@ func (r repoIgnoreTester) IsIgnored(f string, isDir bool) (bool, error) {
 }
 
 func NewRepoIgnoreTester(ctx context.Context, repoRoot string) (IgnoreTester, error) {
-	g, err := NewGitIgnoreTester(ctx, repoRoot)
+	absRepoRoot, err := filepath.Abs(repoRoot)
```

Comment:
still a bit concerned that these need to be absolutified further upstream...

filepath.Abs means 'resolved relative to the process working directory', which seems like a weird thing to do in a low-level library like this